tools/gen_customop_template.py

```diff
@@ -0,0 +1,98 @@
+'''
+Generates the C++ template code for a new custom operator.
+
+Usage: $ python tools\gen_customop_template.py onnx_model_filepath
+'''
+
+import argparse
+import onnx
+import pathlib
+import sys
+
+# Load and check ONNX model
+def load_onnx_model(model_path):
+    model = onnx.load(model_path)
+    onnx.checker.check_model(model)
```

Comment:
Did you verify that the model check passes if there is a custom operator in the model?

Reply:
Yep! Will add a test for the same.